package.json

```diff
@@ -27,6 +27,7 @@
     "browserify": "^16.3.0",
     "chai": "^4.2.0",
     "dirty-chai": "^2.0.1",
+    "json5": "^2.1.3",
```

Comment:
I believe that it's a runtime dependency since we are using it in the runtime code.
We need to check if it's working in a browser environment.

Reply:
What's the easiest way to test, if it is working in a browser environment? 

By the way, I was not able to run the "test:browser", because of the following error:
```
Unable to run tests using Puppeteer Error: Failed to launch the browser process!
.../asciidoctor-kroki/node_modules/puppeteer/.local-chromium/linux-737027/chrome-linux/chrome: error while 
loading shared libraries: libXcursor.so.1: cannot open shared object file: No such file or directory
```

I'm working on Windows with WSL (Windows Subsystem for Linux), which I installed yesterday. I'll probably have to install something there, but I haven't had time to have a closer look at it. The other NPM scripts worked fine.

Reply:
> What's the easiest way to test, if it is working in a browser environment?

We have tests running with Puppeteer/Chrome to make sure that it's working in a browser environment.
You can also build the project and open the page https://github.com/Mogztter/asciidoctor-kroki/blob/master/test/browser/index.html using a local HTTP server (ie. it won't work if you open the page in your browser using the `file://` protocol because CORS)

> By the way, I was not able to run the "test:browser", because of the following error:

Puppeteer/Chrome requires a few system dependencies so you will probably need to install `libxcursor1`. Are you using Ubuntu on WSL? If so you can try: https://github.com/puppeteer/puppeteer/issues/807#issuecomment-436588786

Reply:
I moved JSON5 from "devDependencies" to "dependencies". "test/browser/index.html" in browser and "npm run test:browser" work (Thanks for the hints!). Is there anything else to do regarding the new dependency?

Reply:
No I think it's good.
I will try to checkout your branch in the next days to make sure that everything is working in the browser extension: https://github.com/asciidoctor/asciidoctor-browser-extension